osu.Game/Screens/Play/HUD/LegacyComboCounter.cs

```diff
@@ -164,27 +184,34 @@ private void transformPopOut(int newValue)
         {
             ((IHasText)popOutCount).Text = formatCount(newValue);
 
-            popOutCount.ScaleTo(1.6f);
-            popOutCount.FadeTo(0.75f);
-            popOutCount.MoveTo(Vector2.Zero);
+            popOutCount.ScaleTo(1.56f);
+            popOutCount.FadeTo(0.6f);
 
-            popOutCount.ScaleTo(1, pop_out_duration);
-            popOutCount.FadeOut(pop_out_duration);
-            popOutCount.MoveTo(displayedCountSpriteText.Position, pop_out_duration);
+            popOutCount.ScaleTo(1, big_pop_out_duration);
+            popOutCount.FadeOut(big_pop_out_duration);
         }
 
         private void transformNoPopOut(int newValue)
         {
             ((IHasText)displayedCountSpriteText).Text = formatCount(newValue);
 
+            counterContainer.Size = displayedCountSpriteText.Size;
+
             displayedCountSpriteText.ScaleTo(1);
         }
 
         private void transformPopOutSmall(int newValue)
         {
             ((IHasText)displayedCountSpriteText).Text = formatCount(newValue);
-            displayedCountSpriteText.ScaleTo(1.1f);
-            displayedCountSpriteText.ScaleTo(1, pop_out_duration);
+
+            counterContainer.Size = displayedCountSpriteText.Size;
+
+            // In stable, small pop out always starts from its initial size
+            displayedCountSpriteText.ClearTransforms();
+
+            displayedCountSpriteText.ScaleTo(1).Then()
+                                    .ScaleTo(1.1f, small_pop_out_duration / 2, Easing.InQuad).Then()
+                                    .ScaleTo(1, small_pop_out_duration / 2, Easing.OutQuad);
```

Comment:
actually, on a second check - where's `{In,Out}Quad` coming from? stable uses just `{In,Out}` if i'm looking correctly

Reply:
From analyzing recordings. Pretty sure those are synonyms, at least in osu-framework, but I prefer using `{In,Out}Quad` because it's more descriptive:
https://github.com/ppy/osu-framework/blob/bebd4c04d78674cc9e8cc75b405737700871d28e/osu.Framework/Graphics/Transforms/DefaultEasingFunction.cs#L46-L52

Reply:
I'd rather use `In` and `Out` here to match stable and avoid confusion. Didn't even realise these were synonyms until now.